lib/idx/authenticator/WebauthnEnrollment.ts

```diff
@@ -0,0 +1,29 @@
+import { Authenticator } from './Authenticator';
+import { RemediationValues } from '../remediators';
+
+export interface WebauthnEnrollValues extends RemediationValues {
+  clientData?: string;
+  attestation?: string;
+}
+
+export class WebauthnEnrollment extends Authenticator<WebauthnEnrollValues> {
+  canVerify(values: WebauthnEnrollValues) {
+    const { clientData, attestation } = values;
+    return (!!clientData && !!attestation);
```

Comment:
nit: how about `!!(clientData && attestation)` ?

Reply:
double bang is used to ensure a variable has a value, aka isn't `undefined` or `null`. `!!(clientData && attestation)` is logically not the same, because the inner `(clientData && attestation)` will always return a value, which makes the `!!` redundant.

As written it ensures both `clientData` and `attestation` are non-falsey values, which I think makes sense

Reply:
I though `!!` is used to transform the return value to a boolean value, and `clientData && attestation` can result in a non-boolean value (so `!!` should not be redundant here), maybe I remember something wrong :)